src/instances/Common.ts

```diff
@@ -252,6 +252,31 @@ export async function confirmReaction(
         .appendLine(2);
 
     if (reactInfo.type === "EARLY_LOCATION") {
+        
+        /**
+         * Temporary Nitro Solution
+         * Check if user who reacted to Nitro has the server booster role
+         */
+
+        if(mapKey === "NITRO"){
+
+            //Obtain Nitro role id if present on server
+            var nitroRoleID = ""; //To test on Test Server, initialize nitroRoleID to a role on the server.
```

Comment:
Generally speaking, try to avoid using `var`.

Reply:
Changed to `let`